package/network/services/dnsmasq/files/dhcp.conf

```diff
@@ -15,7 +15,7 @@ config dnsmasq
 	option leasefile	'/tmp/dhcp.leases'
 	option resolvfile	'/tmp/resolv.conf.auto'
 	#list server		'/mycompany.local/1.2.3.4'
-	#option nonwildcard	1
+	option nonwildcard	1 # bind to & keep track of interfaces
```

Comment:
Ok, so I've done that which of course leads to tweaking luci now as well https://github.com/kdarbyshirebryant/luci/commit/17882e86321749c043b3581bd7540a566c2e1e0b

Not done the translations - no idea how.  The other 'fun' aspect - the 'ignoring/including' interface option was dependent on 'non wildcard' - which is really bogus 'cos ignoring or including interfaces is independent of dnsmasq's binding mechanism.  Ah LEDE, changing one thing leads to 6 other changes....across repos. joy.